test/validators/movies/list.test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const Joi = require('joi');
+
+const MovieListValidator = require('../../../lib/validators/movies/list');
+
+describe('movie list validator', () => {
+
+  it('succeeds when all allowed params are included', () => {
+    const name = 'title';
+    const release_year = {
+      gt: 2000,
+      lt: 2017
+    };
+    const payload = { name, release_year };
+    const result = Joi.validate(payload, MovieListValidator);
+
+    expect(result.error).to.be.null;
+  });
+
+  describe('name', () => {
+
+    it('is optional', () => {
+      const payload = {};
+      const result = Joi.validate(payload, MovieListValidator);
+
+      expect(result.error).to.be.null;
+    });
+
+  });
+
+  describe('release_year', () => {
+
+    it('can be an object', () => {
```

Comment:
What about testing the current `.min(1)` (or whatever the final value ends up being) restriction? Currently you don't have any test that guarantees that people cannot search for a movie released in `-1`.